diff --git a/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldImage.vue b/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldImage.vue
index 628810c..c54dc9e 100644
--- a/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldImage.vue
+++ b/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldImage.vue
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import { ref } from 'vue'
+import { ref, toRefs, watchEffect } from 'vue'
 import { Image } from '@unpic/vue'
 import type {
   ConstructUrlProps,
@@ -9,15 +9,18 @@ import type {
 import { pollForProcessingImage } from '@cloudinary-util/util'
 import { useCldImageUrl } from '../composables/useCldImageUrl'
 
-export interface CldImageProps extends ImageOptions {
+export interface CldImageProps extends /* @vue-ignore */ ImageOptions {
+  src: string//新增
+  quality?: number//新增
+  gravity: string//新增
+  crop: any//新增 fill填充 crop裁剪
+  aspectRatio?: number//新增
   loading?: 'eager' | 'lazy'
   fetchPriority?: 'high' | 'low' | 'auto'
-  // Adding below as required props to promote good patterns in developing images
-  alt: string
-  width: string | number
-  height: string | number
+  alt?: string
+  width: number
+  height: number
   config?: ConfigOptions
-  // Unpic props
   layout?: 'constrained' | 'fullWidth' | 'fixed'
   priority?: boolean
   background?: 'auto' | string
@@ -26,10 +29,11 @@ export interface CldImageProps extends ImageOptions {
 const props = defineProps<CldImageProps>()
 
 const { config, ...options } = props
-
+console.log('config', config,)
+console.log('options', options)
 const { url } = useCldImageUrl({ options, config } as ConstructUrlProps)
 
-const transformUrl = () => {
+ const transformUrl = () => {
   const options = {
     ...props,
   }
@@ -44,41 +48,44 @@ const transformUrl = () => {
       : options.height
 
   const { url } = useCldImageUrl({ options, config } as ConstructUrlProps)
+  console.log('transformUrl 返回的新 url', url)
 
   return url
-}
+ }
 
+/* ===== 5. 错误处理 ===== */
 const imgKey = ref('image-key')
 
 const handleError = async (payload: Event) => {
   const eventPayload = payload.target as EventTarget & { src: string }
+
   const result = await pollForProcessingImage(eventPayload)
 
-  if (
-    typeof result.error === 'string'
-    && process.env.NODE_ENV === 'development'
-  ) {
-    console.error(
-      `[CldImage] Failed to load image ${props.src}: ${result.error}`,
-    )
+  if (typeof result.error === 'string' && process.env.NODE_ENV === 'development') {
+    console.error(`[CldImage] 加载失败: ${result.error}`)
   }
 
-  if (result.success) imgKey.value = `${imgKey.value}-${Math.random()}`
+  if (result.success) {
+    imgKey.value = `${imgKey.value}-${Math.random()}`
+    // console.log('轮询成功，强制重新渲染，新 key', imgKey.value)
+  }
 }
 </script>
 
 <template>
+  <!-- 必须设置 width height 才能正常 不报错：不要乱改 -->
   <Image
     :key="imgKey"
-    :src="url"
-    :layout="layout || 'constrained'"
     :width="width"
     :height="height"
+    :src="url"
+    :layout="layout || 'constrained'"
     v-bind="$attrs"
     :alt="alt"
     :priority="priority"
     cdn="cloudinary"
     :transformer="transformUrl"
     @error="handleError"
+    background="#ccc"
   />
 </template>
\ No newline at end of file
diff --git a/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldOgImage.vue b/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldOgImage.vue
index 77b5af4..b12a5dc 100644
--- a/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldOgImage.vue
+++ b/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldOgImage.vue
@@ -12,7 +12,7 @@ const OG_IMAGE_HEIGHT = 1254
 
 const { currentRoute } = useRouter()
 
-export type CldOgImageProps = CldImageProps & {
+export type CldOgImageProps = /* @vue-ignore */ CldImageProps & {
   excludeTags?: Array<string>
   twitterTitle?: string
   alt?: string
diff --git a/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldUploadWidget.vue b/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldUploadWidget.vue
index 0910b1d..77a81ce 100644
--- a/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldUploadWidget.vue
+++ b/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldUploadWidget.vue
@@ -15,7 +15,7 @@ import { triggerOnIdle } from '../util/triggerOnIdle'
 import { useRuntimeConfig } from '#imports'
 
 export interface CldUploadWidgetProps
-  extends GenerateUploadWidgetResultCallback {
+  extends /* @vue-ignore */ GenerateUploadWidgetResultCallback {
   options?: GetUploadWidgetOptions
   signatureEndpoint?: URL | RequestInfo
   uploadPreset?: string
diff --git a/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldVideoPlayer.vue b/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldVideoPlayer.vue
index 49c5857..b5d6ab5 100644
--- a/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldVideoPlayer.vue
+++ b/node_modules/@nuxtjs/cloudinary/dist/runtime/components/CldVideoPlayer.vue
@@ -29,7 +29,7 @@ export interface CloudinaryVideoPlayerOptionsColors {
   text?: string
 }
 
-export type CldVideoPlayerProps = GetVideoPlayerOptions & {
+export type CldVideoPlayerProps = /* @vue-ignore */ GetVideoPlayerOptions & {
   autoPlay?: boolean
   autoplayMode?: 'never' | 'always' | 'on-scroll'
   className?: string
diff --git a/node_modules/@nuxtjs/cloudinary/dist/runtime/composables/useCldImageUrl.mjs b/node_modules/@nuxtjs/cloudinary/dist/runtime/composables/useCldImageUrl.mjs
index 26f36cb..8e9a59f 100644
--- a/node_modules/@nuxtjs/cloudinary/dist/runtime/composables/useCldImageUrl.mjs
+++ b/node_modules/@nuxtjs/cloudinary/dist/runtime/composables/useCldImageUrl.mjs
@@ -3,8 +3,8 @@ import nuxtPkg from "nuxt/package.json";
 import pkg from "../../../package.json";
 import { useRuntimeConfig } from "#imports";
 export const useCldImageUrl = (props) => {
-  if (!props.options.src)
-    console.error("`[@nuxtjs/cloudinary]`: Property `src` is missing");
+  // if (!props.options.src)
+  //   console.log("`[@nuxtjs/cloudinary]`: Property `src` is missing123");
   const moduleConfig = useRuntimeConfig().public.cloudinary;
   const cldCloudName = props.config?.cloud?.cloudName || moduleConfig.cloud?.cloudName || moduleConfig?.cloudName;
   if (!cldCloudName)
@@ -35,6 +35,7 @@ export const useCldImageUrl = (props) => {
       }, props.analytics)
     };
   }
+  console.log('cldOptions', constructCloudinaryUrl(cldOptions))
   return {
     url: constructCloudinaryUrl(cldOptions)
   };
